docs/sample-solutions/windows-server/healthcare-multitier-app.md

```diff
@@ -0,0 +1,111 @@
+author: iainfoulds
+ms.date: 06/26/2018
+---
+# Deploy a secure and scalable multi-tier web application on Windows Server for a healthcare solution
+
+This sample solution is applicable to healthcare organizations that have a need to secure multi-tier applications. In this solution, a front-end ASP.NET application securely connects to a protected back-end Microsoft SQL Server cluster.
+
+Example application scenarios include running operating room applications, patient appointments and records keeping, or prescription refills and ordering. Traditionally, organizations had to maintain legacy on-premises applications and services for these scenarios. With a secure way and scalable way to deploy these Windows Server applications in Azure, organizations can modernize their deployments are reduce their on-premises operating costs and management overhead.
+
+## Potential use cases
+
+You should consider this solution for the following use cases:
+
+* Modernizing application deployments in a secure cloud environment.
+* Reducing legacy on-premises application and service management.
+* Improving patient healthcare and experience with new application platforms.
+
+## Architecture diagram
+
+The solution diagram below is an example of this solution:
+
+![Architecture overview of the Azure components involved in multi-tier Windows Server application for healthcare][architecture]
+
+## Architecture
+
+This solution covers a multi-tier healthcare application that uses ASP.NET and Microsoft SQL Server. The data flows through the solution as follows:
+
+1. Users access the front-end ASP.NET healthcare application through an Azure Application Gateway.
+2. The Application Gateway distributes traffic to VM instances within an Azure virtual machine scale set.
+3. The ASP.NET application connects to Microsoft SQL Server cluster in a back-end tier via an Azure load balancer. These backend SQL Server instances are in a separate Azure virtual network, secured by network security group rules that limit traffic flow.
+4. The load balancer distributes SQL Server traffic to VM instances in another virtual machine scale set.
+5. A VM that runs Windows Server 2016 within the back-end tier virtual network acts as a file share witness for the SQL Server cluster.
+
+### Components
+
+* [Azure Application Gateway][appgateway-docs] is a layer 7 web traffic load balancer that is application-aware and can distribute traffic based on specific routing rules. App Gateway can also handle SSL offloading for improved web server performance.
+* [Azure Virtual Network][vnet-docs] allows resources such as VMs to securely communicate with each other, the Internet, and on-premises networks. Virtual networks provide isolation and segmentation, filter and route traffic, and allow connection between locations. Two virtual networks are used in this solution to provide a common demilitarized zone (DMZ) and isolation of the application components.
```

Comment:
How do two VNets form a DMZ, is there a firewall? Are they peered?

Reply:
Vnet peering. Added sentence on how they'd connect. The template does hint at this, but doesn't implement due to circular dependency, I think - https://github.com/iainfoulds/architecture-center/blob/windowsserver/docs/sample-solutions/windows-server/templates/healthcare-multitier-app.json#L180

Reply:
But then how is that a DMZ? If they are peered, traffic can go between them.